CONTRIBUTING.md

```diff
@@ -70,9 +70,12 @@ Most components should use the compose framework as it offers the comprehensive
 ### Adding a new test for your component to the test app
 
 1. In FluentTester (our test app), we'll need to add an test component. The test components are located under `fluentui-react-native/apps/fluent-tester/src/FluentTester/TestComponents`.
-1. Make a subdirectory for your new component at this path, as well as adding an entry for your test in `Tests.tsx` and `Tests.<platform>.tsx`
+1. Make a subdirectory for your new component at this path, as well as adding an entry for your test in `testPages.tsx` and `testPages.<platform>.tsx`
 1. Create your tests in this new subdirectory. Pattern matching off an existing control's tests will greatly help.
-1. If you're component contains a native module, you will also need to add the path to your component's `podspec` (that we created earlier) in your test app's Podfile. This extra step is due to the fact that FluentTester has separate platform test apps that each share a common JS package. The react native community CLI does not support [autolinking transitive dependencies](https://github.com/react-native-community/cli/issues/1347), so we need to add it manually here.
+1. If your component contains a native module, you will also need to add the path to your component's `podspec` (that we created earlier) in your test app's Podfile. This extra step is due to the fact that FluentTester has separate platform test apps that each share a common JS package. The react native community CLI does not support [autolinking transitive dependencies](https://github.com/react-native-community/cli/issues/1347), so we need to add it manually here.
+1. Add your new module as a dependency in `apps/fluent-tester/package.json` and run `yarn && yarn build` from the root folder.
+   1. If your component has native apple code, run `pod install`.
```

Comment:
nit: Run pod install on the corresponding macOS or iOS test app.

Reply:
This is probably the second document that a newbie will read in this repo.  At that point I do not expect it will be clear what "on the corresponding macOS or iOS test app" means.  I would like this to be more explicit, e.g. "from the macOS or iOS test app `src` folder (`apps/macos/src` or `apps/ios/src` respectively)."

Reply:
That makes sense. I personally first ran the test apps first which requires you to run `pod install`, but it is nice to clarify to ensure everyone understands what to do.